src/format/html/format-html.ts

```diff
@@ -75,11 +76,11 @@ export function htmlFormat(
     createHtmlFormat(figwidth, figheight),
     {
       metadataFilter: (metadata: Metadata) => {
-        if (!!metadata[kMinimal] && metadata[kTheme] === undefined) {
-          return {
-            ...metadata,
-            theme: "none",
-          };
+        if (!metadata[kMinimal]) {
+          return mergeConfigs(metadata, {
+            [kFigResponsive]: false,
+            theme: metadata[kTheme] === undefined ? "none" : metadata[kTheme],
+          });
```

Comment:
It will now be set to TRUE by default except when minimal is used. There is no more Bootstrap conditional as discussed